### DISCUSSION TITLE
Customizing window movement behavior

### DISCUSSION THREAD

**Discussion by m1cahg** [i3/i3#5027] [Q&A]
I don't know all the terminology so hopefully you can understand this.

I use tabbed layout, and I want the behavior when I press MOD+down on a tabbed window for that window to take up half of the tabbed group and not half of the entire monitor.

https://imgur.com/a/JOWSjKK here are pictures this should explain it

thanks for any and all help!

**Comment by orestisfl** [comment]
If I understand the layout correctly, you need to first switch to the vertical layout ($mod+v) and then $mod+down

**Reply by m1cahg** [reply]
Thanks, however I like using tabs and want that functionality to stay but is there a way to change just the one interaction I mentioned?

**Reply by orestisfl** [reply]
Switch a tab to vertical and move a window to it with left/right

![image](https://user-images.githubusercontent.com/5778622/191572972-79edb7c3-91ba-4f95-b65c-12ec0bd2dc01.png)
